### ISSUE TITLE
[1.18.1] Netherrocks Furnaces Incompatibilities

### ISSUE THREAD

**Issue by Cicopath** [Sinhika/Netherrocks#19]
Forge: 1.18.1-39.1.2
Netherrocks: 1.18.1-4.0.1.0

The Nether Furnace, Nether Smoker, and Nether Blast Furnace are incompatible with two mods: Polymorph and Botania.

_Polymorph_
Polymorph is a mod that fixes recipe conflicts. When ores of the same type are put in a furnace, like Immersive Engineering and Occultism silver ore, Polymorph allows you to choose which silver you want. However, this does not work with the Netherrocks furnaces.

_Botania_
Botania has a flower—the Exoflame—which, "powers," furnaces when fed mana, thus no longer requiring fuel in the furnaces. Unfortunately, the Exoflame does not power Netherrock furnaces.

I am unsure who exactly adds the compatibility, but I thought it would be more important to bring it up here considering the couple of incompatibilities.

**Comment by Sinhika** [comment]
Note: most of these incompatibilities are to be expected, as the Nether Furnaces are not related to vanilla furnaces at all, internally. They are their own breed of machines, and do not descend from the vanilla furnace classes.

**Comment by Cicopath** [comment]
Of course; understandable. I thought it would be neat to see some mod compatibility. (Except the recipe conflict incompatibility. I believe that would be dire to fix considering how important some materials are for their respective mods.)
